src/mapper.ts

```diff
@@ -89,13 +89,29 @@ const mapRss2ToJsonFeed = (rss: RSS2): JsonFeed => {
 		home_page_url: channel.link,
 		items,
 		author,
-		hubs
+		hubs,
 	} as JsonFeed;
 };
 
 const mapAtomToJsonFeed = (atom: Feed): JsonFeed => {
 	const items: JsonFeedItem[] = atom.entries.map((entry) => {
 		let author;
+		let url: string | undefined;
```

Comment:
Please provide matching tests in mapper_test.ts.

Micro-optimization though (optional): If you replace your three if-statements with an else if but check the fields in this order: feedburner-url, href and then id we would save two if-calls for the feedburner urls.